src/DataGrid.tsx

```diff
@@ -1027,6 +1027,24 @@ function DataGrid<R, SR>({
       )}
       {rows.length === 0 && EmptyRowsRenderer ? <EmptyRowsRenderer /> : (
         <>
+          {/*
+            Ideally the focus should be set on the selected cell
+            (https://www.w3.org/TR/wai-aria-practices-1.2/#kbd_roving_tabindex)
+            but there are a few issues with this approach
+            - onKeyDown/useLayoutEffect on the cell is not fast enough and grid
+              looses focus if tab is pressed repeatedly
```

Comment:
Would it work if we managed focus in a `ref` callback instead of `useLayoutEffect`? 🤔

Reply:
Not sure. We can check